### PR TITLE
Fixed some spelling mistakes

### PR DESCRIPTION
**Pull request by yenicapotediaz**
And other stuff that made no sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Additionally, you can deploy some or all of these charts individually with the f
 - [ElasticSearch Curator](https://github.com/samsung-cnct/chart-curator)
 - [Kibana](https://github.com/samsung-cnct/chart-kibana)
 
-*These charts should work on any Kubernetes cluster.*
+*These charts should work on any Kubernetes cluster*
```
Comment (joejulian) [inline README.md]: Please put the punctuation back here.

Comment (philoserf) [reply README.md]: not disagreeing or agreeing. just think why is useful.

Comment (davidewatson) [reply README.md]: Sentences end with periods? ;)

Comment (yenicapotediaz) [reply README.md]: Fine, I'll add it back.

Comment (philoserf) [reply README.md]: don't be cowed by imperative language  @yenicapotediaz

evil grin

**Comment by guineveresaenger** [comment]
You people are hilarious.

**Pull request merged**